package.json

```diff
@@ -109,6 +110,9 @@
     "main": "lib/ember-addon-main.js",
     "configPath": "tests/dummy/config"
   },
+  "resolutions": {
+    "ember-cli-htmlbars": "^6.1.0"
```

Comment:
I needed to add this because `ember-styleguide` depends on an earlier version of `ember-cli-htmlbars` and that prevents the bug from affecting that addon. 

It's an interesting potential mitigation for anyone with this problem, make sure your **addon** doesn't depend on the latest ember-cli-htmlbars 🤔

Reply:
fixed this using `link:.` as the resolution